src/dashboard/components/CashFlowGrid/CashFlowGrid.tsx

```diff
@@ -0,0 +1,92 @@
+import { Group, Paper, SimpleGrid, Text } from '@mantine/core';
+import {
+  IconUserPlus,
+  IconDiscount2,
+  IconReceipt2,
+  IconCoin,
+  IconArrowUpRight,
+  IconArrowDownRight,
+  IconEqual,
+  IconCash,
+  IconPigMoney,
+  IconReportMoney,
+} from '@tabler/icons-react';
+import classes from './CashFlowGrid.module.css';
+import { formatCurrency, formatPercentage } from '@/helper/formatter';
+
+enum IconsKey {
+  user = 'user',
+  discount = 'discount',
+  receipt = 'receipt',
+  coin = 'coin',
+  cash = 'cash',
+  pig = 'pig',
+  report = 'report',
+}
+
+const icons = {
+  [IconsKey.user]: IconUserPlus,
+  [IconsKey.discount]: IconDiscount2,
+  [IconsKey.receipt]: IconReceipt2,
+  [IconsKey.coin]: IconCoin,
+  [IconsKey.cash]: IconCash,
+  [IconsKey.pig]: IconPigMoney,
+  [IconsKey.report]: IconReportMoney,
+};
+
+export interface TransactionStatsProps {
+  statistics: {
```

Comment:
Seems like this is going to be a re-usable type later on so maybe have a explicit type

Reply:
Had a bunch of typing errors when I tried creating a TransactionStats type, so I just gave up lol